fern/pages/api-definition/fern-definition/websockets.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: WebSockets in the Fern Definition 
+description: Learn how to define WebSockets in the Fern Definition
+---
+
+In Fern, you can specify WebSockets in your API definition. The WebSockets will be included in both the generated SDKs and the API documentation. 
```

Comment:
Up to add a sentence that helps a reader understand "what's a WebSocket?"

such as: WebSockets are used for real-time, event-driven communication between clients and server